pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -23,13 +23,21 @@ import (
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
+// ValidateBindingName is the validation function for Binding names.
+var ValidateBindingName = apivalidation.NameIsDNSSubdomain
```

Comment:
EQ: Why the limitation on the chars being used? If there are kube rules that all names must conform to rfc2119 then ok, but I am kind of curious why there's a limitation when things can be escaped.